src/main/java/seedu/address/logic/commands/ClearCommand.java

```diff
@@ -3,20 +3,22 @@
 import static java.util.Objects.requireNonNull;
 
 import seedu.address.model.AddressBook;
+import seedu.address.model.AppointmentBook;
 import seedu.address.model.Model;
 
 /**
- * Clears the address book.
+ * Clears the sample data in the appointment book.
  */
 public class ClearCommand extends Command {
 
     public static final String COMMAND_WORD = "clear";
```

Comment:
should there be two seperate commands for property and appointment?

Reply:
I think it should be one command for both property book and address book? Which one is better?

Reply:
Maybe something like `clear appointment` and `clear property`? As well as a `clear all` to clear everything at once?

Reply:
Ok, I got what you mean. So split the command into 3